src/openrct2/peep/Peep.h

```diff
@@ -529,6 +529,14 @@ enum PeepRideDecision
     PEEP_RIDE_DECISION_THINKING = 1 << 2,
 };
 
+// Flags used by PeepThoughtToActionMap
+enum PeepThoughtToAction : uint8_t
+{
+    PEEP_THOUGHT_ACTION_FLAG_1 = (1 << 0),
+    PEEP_THOUGHT_ACTION_FLAG_2 = (1 << 1),
+    PEEP_THOUGHT_ACTION_FLAG_4 = (1 << 2),
```

Comment:
Flag 3. 

We have been a bit inconsistent in the codebase with what we call flags. We sometimes zero index them so that they are the amount we want to left shit. Other times we have one indexed them so that its the number - 1 of the amount left shited.